src/main/java/net/revelc/code/formatter/xml/lib/CommentFormatter.java

```diff
@@ -0,0 +1,52 @@
+package net.revelc.code.formatter.xml.lib;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class CommentFormatter {
+
+    private static final Pattern ORIGINAL_INDENT_PATTERN = Pattern.compile("^(?<indent>[\\s\\t]*)-->");
```

Comment:
I think `\s` will match on space or tab, so having a character class (defined with square brackets) that includes `\s` *and* `\t` doesn't make sense to me.

I think you can have an equivalent:

```suggestion
    private static final Pattern ORIGINAL_INDENT_PATTERN = Pattern.compile("^(?<indent>\\s*)-->");
```